Improve `otherwise` docs (#4814)
tenzir/tenzir: web/docs/tql2/functions/otherwise.md

Returns a `fallback` value if `primary` is `null`.

```tql
otherwise(primary:any, fallback:any) -> any
```

## Description

The `otherwise` function evaluates its arguments and replaces `primary` with
`fallback` where `primary` would be `null`.

### `primary: any`

The expression to return if not `null`.

### `fallback: any`

> The expression to return if `primary` evaluated to `null`. Currently, must be
> evaluate to the same type as `primary`.

Reviewer: "must be evaluate to"
```suggestion
The expression to return if `primary` evaluates to `null`. It must have the same
type as `primary`.
```
